src/App.test 2.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Tic Tac Toe Application', () => {
+
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+
+    ReactDOM.render(<App />, div);
```

Comment:
I would favor using `mount` or `shallow` from `Enzyme` instead of `ReactDOM.render` in tests.